models/parking.go

```diff
@@ -0,0 +1,26 @@
+package models
+
+import (
+	"gorm.io/gorm"
+	"time"
+)
+
+// Parking model
+type Parking struct {
+	gorm.Model
+	CarID       uint      `json:"car_id"`
```

Comment:
why are we using carid in parking model?

Reply:
It will create a foreign key relationship b/w parking and car models,this allows us to retrieve all the parking records for a specific car,as one car can have multiple parking records

Reply:
remove this, we are not using a relational DB your logic has to compensate for that